sql/moz-fx-data-shared-prod/mozilla_vpn_derived/login_flows_v1/metadata.yaml

```
@@ -0,0 +1,17 @@
+friendly_name: Mozilla VPN FxA Login Flows
+description: >
+  A list of users that attempted to login to Mozilla VPN.
```

This makes it sound like they were existing VPN users, logging back in, whereas we want to study prospective VPN users, logging in to FxA as part of the VPN signup flow.

Is this accurate?
```suggestion
  A list of users that attempted to login to FxA, to sign up for Mozilla VPN.
```

Or should it be
```suggestion
  A list of users that attempted to login to FxA, to sign up for or log in to Mozilla VPN.
```

---

this first suggestion lgtm

---

oh, actually thinking it through the second one is more accurate